test/claim-issuers/claim-to.test.ts

```diff
@@ -0,0 +1,166 @@
+import { expect } from "chai";
+import { ethers } from "hardhat";
+import { BaseContract, ContractFactory } from "ethers";
+import { HardhatEthersSigner } from "@nomicfoundation/hardhat-ethers/signers";
+import { deployIdentityFixture } from "../fixtures";
+import { loadFixture } from "@nomicfoundation/hardhat-network-helpers";
+
+
+interface IIdentity extends BaseContract {
```

Comment:
please copy the same structure for your test file as what we used for all other tests, we need to keep the test files coherent together.